### PR TITLE
Adds DeltaStation's emergency shuttle to the shuttle file list

### PR DESCRIPTION
**Pull request by Okand37**
Adds DeltaStation's emergency shuttle to the shuttle file list. First proper PR to add anything to the game, lets see how it goes. I may very well have done this wrong!

:cl: Okand37
add: DeltaStation's emergency shuttle
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
yuge

**Comment by Supermichael777** [comment]
Should call it big bertha

**Comment by ExcessiveUseOfCobblestone** [comment]
the escape shuttle is a station in itself jesus

**Comment by Jordie0608** [comment]
This PR has commits including Deltastation itself, not just the shuttle file.
Also you'll want to add a datum for this shuttle to `https://github.com/tgstation/tgstation/blob/master/code/datums/shuttles.dm` to have it actually usable.

**Comment by Arianya** [comment]
What price point did you see for this shuttle? Currently the highest cost (I think?) is the Lepton Violet which is sizably smaller then this one. Admittedly asteroid is probably larger, but its also a ~meme shuttle~

**Comment by Okand37** [comment]
Oh right, I should fix this today shouldn't I! Its a large shuttle as its meant to contain a highpopulation for, respectably, a highpopulation station. If I am unmistaken, however, its roughly 18x30 and should fit into most docks (will need to put the docking point down an airlock so that it works with boxstation ports.)
Will make sure to fix the commit and remove the magnetic pull that is dragging the station along with the shuttle into the PR.

I'll probably place it roughly around boxstation/metastation's shuttles but slightly higher, as it is just a standard station 'usable shuttle' but you also need to weigh in the size of what it can carry and obviously at that its slight edge on durability just due to sheer mass.

**Comment by Okand37** [comment]
Right, fixing this now. It'll be placed at 7500 credits, as box/meta are only around 2000 or so.

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -206,3 +206,9 @@
 /datum/map_template/shuttle/cargo/birdboat
 	suffix = "birdboat"
 	name = "supply shuttle (Birdboat)"
+/datum/map_template/shuttle/emergency/delta
+	suffix = "airless"
```
Comment (Jordie0608) [inline code/datums/shuttles.dm]: `suffix = "delta"`

**Review by Jordie0608 — CHANGES_REQUESTED** [review]

**Comment by Jordie0608** [comment]
I forgot I can just change it myself.

**Review by Jordie0608 — APPROVED** [review]

**Comment by jughu** [comment]
I want this to crash into the station

**Comment by Okand37** [comment]
Works for me.

**Pull request merged**